dll/win32/comctl32/edit.c

```diff
@@ -3691,7 +3694,10 @@ static void EDIT_WM_SetFocus(HTHEME theme, EDITSTATE *es)
     if (!(es->style & ES_NOHIDESEL))
         EDIT_InvalidateText(es, es->selection_start, es->selection_end);
 
-    CreateCaret(es->hwndSelf, 0, 1, es->line_height);
+#ifdef __REACTOS__
+    SystemParametersInfo(SPI_GETCARETWIDTH, 0, &es->dwCaretWidth, 0);
+#endif
+    CreateCaret(es->hwndSelf, NULL, es->dwCaretWidth, es->line_height);
```

Comment:
Some changes are guarded with `__REACTOS__` but some are not... this is not clear (same for other places)

Reply:
@binarymaster Thanks for your review